src/Components/Form/Form.js

```diff
@@ -69,8 +86,15 @@ function Form({ contacts, handleEtaChange, userInfo }) {
     e.preventDefault();
   }
 
+
+
   return (
-    <form className='trip-form' onSubmit={handleSubmit}>
+    <form className='trip-form' onSubmit={handleSubmit} >
+    {valid && <p>Complete form fields with Valid Data</p>}
```

Comment:
Error handling looks solid.  Where exactly does the message render?  Does this appear within the form?